### ISSUE TITLE
documentation tiny bug compass

### ISSUE THREAD

**Issue by nielsswinkels** [Microsoft/pxt-microbit#717]
on https://makecode.microbit.org/reference/input/compass-heading

it says it returns values from 0 to 360. Shouldn't that be from 0 to 359?

**Comment by ganicke** [comment]
If ``bearing`` is never exactly `0.0` then ``heading`` will be `0` thru `359`. I assume it never will read exactly `0.0`. I'll correct the range in the doc. Thanks.

https://github.com/lancaster-university/microbit-dal/blob/2cff906f019944ae15d532157c1301c940f67862/source/drivers/MicroBitCompass.cpp#L333

```cpp
/**
  * Calculates a non-tilt compensated bearing of the device.
  */
int MicroBitCompass::basicBearing()
{
    updateSample();

    float bearing = (atan2((double)(sample.y - average.y),(double)(sample.x - average.x)))*180/PI;

    if (bearing < 0)
        bearing += 360.0;

    return (int)(360.0 - bearing);
}
```

**Comment by ganicke** [comment]
Fixed in #721
